backend/session/direct-freebsd.c

```diff
@@ -0,0 +1,253 @@
+#include <errno.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <signal.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <termios.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include <dev/evdev/input.h>
+#include <sys/kbio.h>
+#include <sys/consio.h>
+#include <wayland-server.h>
+#include <wlr/backend/session/interface.h>
+#include <wlr/util/log.h>
+#include "backend/session/direct-ipc.h"
+
+enum { DRM_MAJOR = 226 };
```

Comment:
Are you sure this is correct on FreeBSD?
It was my understanding that this number is linux-specific.

Reply:
Why would it be different? The drm/kms subsystem is copied from linux :) We don't use major/minor numbers generally, major is 0 for everything else.

Reply:
I don't think you can get the same drm major value on freebsd. At least not with drm-next kernel iirc. In wlc I think I just skip that check.

Reply:
Oh — the check currently does the `setmaster` stuff. I guess I can remove some code :)